rakelib/plugins-metadata.json

```diff
@@ -441,6 +441,14 @@
     "default-plugins": true,
     "skip-list": false
   },
+  "logstash-integration-internal": {
+    "default-plugins": false,
+    "skip-list": true
+  },
+  "logstash-integration-zeromq": {
+    "default-plugins": false,
+    "skip-list": true
+  },
```

Comment:
Note to self @karenzone: Currently these two plugins are being skipped because of logic in docs tooling.  I'm merging these changes to the skiplist now, even though necessary changes in [docs tooling](https://github.com/elastic/docs-tools/pull/107) are still under consideration.